file_paths.md

```diff
@@ -289,6 +290,9 @@ running as root and non-root, e.g. `puppet.conf` is always
     ~/.puppetlabs/var/log                 # :logdir                     ~/.puppet/var/log
     ~/.puppetlabs/opt/facter/facts.d      # n/a                         ~/.facter/facts.d
 
+On Windows, when not running on the SYSTEM account
```

Comment:
You need a newline after this line for markdown to properly format the next line.